app/Main.hs

```diff
@@ -142,8 +140,8 @@ main = do setLocaleEncoding utf8
             >>= whenCore (\ctx -> (execStr "Internal" "(deftype ExecutionMode Repl Build Install Run Check)") ctx
                          >>= (execStr "" "(doc ExecutionMode \"Represents the mode the Carp compiler is executing in.\")")
                          >>= setExecutionMode execMode
-                         >>= (execStr "" executionModeDoc)
-                         >>= (execStr "" executionModeDynamicDoc))
```

Comment:
Can’t the dynamic part be offloaded into `StartingEnv.hs`?

Reply:
I think theoretically it could, but I'm a bit reluctant since then we'd have to add logic to pull out the execmode from the starting env's context, and I kinda feel the current impl is slightly simpler